src/languages/es.json

```diff
@@ -0,0 +1,118 @@
+{
+    "common": {
+        "copy": "Copiar",
+        "open": "Abrir",
+        "send": "Enviar",
+        "cancel": "Cancelar",
+        "account": "Cuenta",
+        "amount": "Cantidad",
+        "total_amount": "Cantidad Total",
+        "your_account": "Mi Cuenta",
+        "public_address": "Dirección Pública",
+        "secret_seed": "Semilla Secreta",
+        "receive": "Recibir",
+        "account_address": "Dirección de la cuenta",
+        "generator": "Generador",
+        "close": "Cerrar",
+        "transaction_fee": "Comisión de la transacción",
+    },
+    "header": {
+        "see_manual": "Guía de usuario",
+        "pdf_path": "https://boscoin.io/docs/wallet-guide-eng.pdf",
+    },
+    "welcome_view": {
+        "title": "Monedero BOScoin",
+        "title_description": "Crea tu semilla, Gestiona tu monedero, Envía y recibe BOScoins.",
+        "button_make": "Generar llave",
+        "create_account": "Crear cuenta nueva de TestNet",
+        "button_open": "Abrir tu monedero",
+        "guide_title": "Instrucciones para mandar BOScoin con tu cuenta",
+        "guide": {
+            "step1_1": "Crea una nueva llave por ",
+            "step1_2": " botón de arriba",
+            "step2": "Escriba abajo las7 dos llaves y guárdelas de manera segura (¡Muy Importante!)",
+            "step3_1": "Abre tu cuenta por ",
+            "step3_2": " botón de arriba",
+            "step4": "Introduce la semilla de tu dirección en el cuadro blanco",
+            "step5_1": "Presiona ",
+            "step5_2": " el botón y verás tu cuenta",
+            "step6_1": "Presiona ",
+            "step6_2": " el botón e introduce la dirección pública del destinatario en el cuadro blanco",
+        },
+        "error": {
+            "fail_create_account": "Fallo al crear la cuenta."
+        },
+        "test_mode": "Esto es un<br><strong>Monedero web de TEST!</strong>",
+    },
+    "login_view": {
+        "header": "Introduce tu semilla",
+        "guide_line_1": "Por favor, introduce tu semilla secreta para acceder a tu cuenta.",
+        "guide_line_2": "Por favor, guarda la semilla secreta de forma segura, de lo contrario la seguridad de tu monedero podría ser comprometida.",
+        "error": {
+            "invalid_secret_seed": "Semilla secreta inválida o balance insuficiente (1 BOScoin mínimo)."
+        },
+    },
+    "send_coin": {
+        "input_recipient_address": "Introduce la dirección pública del destinatario",
+        "input_amount": "Introduce la cantidad que deseas enviar",
+        "transaction_fee": "Comisión de la transacción",
```

Comment:
What about modifying this with [comisiones de transacción]?

Transaction fee translation reference
  https://blog.bit2me.com/es/explorador-de-blockchain-transacciones/